moveos/moveos-stdlib/src/natives/moveos_stdlib/raw_table/mod.rs

```diff
@@ -630,7 +630,7 @@ fn native_drop_unchecked_box(
     args: VecDeque<Value>,
 ) -> PartialVMResult<NativeResult> {
     assert_eq!(args.len(), 1);
-
+    //TODO remove the droped table from the table_data
```

Comment:
The bug Is it not to check the table is empty?

Reply:
The `drop_unchecked` is `test_only`, so it does not affect the transaction result. I tried to remove the useless `drop_unchecked` function and got that we need to remove the dropped table from the runtime TableData if we want to correct the Table runtime cache.  

Maybe using the dropped table again can produce a bug in the unit tests.